docs/ProductionChecklistForScalarDBCluster.md

```diff
@@ -0,0 +1,140 @@
+# Production checklist for ScalarDB Cluster
+
+This checklist provides recommendations when deploying ScalarDB Cluster in a production environment.
+
+## Before you begin
+
+In this checklist, we assume that you are deploying ScalarDB Cluster on a managed Kubernetes cluster, which is recommended.
+
+## Production checklist: ScalarDB Cluster
+
+The following is a checklist of recommendations when setting up ScalarDB Cluster in a production environment.
+
+### Number of pods and Kubernetes worker nodes
+
+To ensure that the Kubernetes cluster has high availability, you should use at least three worker nodes and deploy at least three pods spread across the worker nodes. You can see the [sample configurations](../conf/scalardb-cluster-custom-values-indirect-mode.yaml) of `podAntiAffinity` for making three pods spread across the worker nodes.
+
+{% capture notice--info %}
+**Note**
+
+If you place the worker nodes in different availability zones (AZs), you can survive an AZ failure.
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
```

Comment:
According to the discussion, we recommend:

* Use at least three nodes (three zones is not necessary).
* Use at least three pods.

And, we add the `you can survive an AZ failure if you use three zones` as a note.

Reply:
I see. I didn't know that term. Thank you!